lib/matplotlib/axes/_base.py

```diff
@@ -2366,17 +2370,19 @@ def _update_patch_limits(self, patch):
         p = patch.get_path()
         vertices = p.vertices if p.codes is None else p.vertices[np.isin(
             p.codes, (mpath.Path.CLOSEPOLY, mpath.Path.STOP), invert=True)]
-        if vertices.size > 0:
-            xys = patch.get_patch_transform().transform(vertices)
-            if patch.get_data_transform() != self.transData:
-                patch_to_data = (patch.get_data_transform() -
-                                 self.transData)
-                xys = patch_to_data.transform(xys)
```

Comment:
note that `get_patch_transform() + get_data_transform() == get_transform() == patch_trf`